flow-typed/box-ui-elements.js

```diff
@@ -671,3 +671,15 @@ type ErrorContextProps = {
         origin: ErrorOrigins,
     ) => void,
 };
+
+type ClassificationInfo = {
+    $canEdit: boolean,
```

Comment:
Everything with $ is generic for metadata instance, so you can be like MetadataInstance & ClassificationInfo

Reply:
Like so, with new $props going to MetadataInstance?

```
type ClassificationInfo = {
    Box__Security__Classification__Key?: string
} & MetadataInstance;
```